.github/workflows/release.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       GITHUB_ACTOR: ${{ secrets.ATALA_GITHUB_ACTOR }}
       GITHUB_TOKEN: ${{ secrets.ATALA_GITHUB_TOKEN }}
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+      # New JDK version makes 'localhost' lookup on linux return ipv6.
+      # Our test containers are on ipv4. We need to make 'localhost' resolve to ipv4.
+      _JAVA_OPTIONS: -Djava.net.preferIPv4Stack=true
```

Comment:
What is the mechanism of propagating `_JAVA_OPTIONS` with an underscore?

Reply:
It's from my local setup, but normal `JAVA_OPTIONS` should also work. Let me try.